apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.RetryConformanceTests/Google.Cloud.Storage.V1.RetryConformanceTests.csproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+	<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFrameworks>netcoreapp3.1;net462</TargetFrameworks>
+		<TestTargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.1</TestTargetFrameworks>
+		<IsPackable>false</IsPackable>
+		<NoWarn>1701;1702;1705;xUnit2004;xUnit2013;AD0001</NoWarn>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+		<PackageReference Include="xunit" Version="2.4.1" />
+		<PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+			<PrivateAssets>all</PrivateAssets>
+		</PackageReference>
+		<PackageReference Include="coverlet.collector" Version="3.1.0">
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+			<PrivateAssets>all</PrivateAssets>
+		</PackageReference>
+	</ItemGroup>
```

Comment:
Some of the dependencies here are outdated with respect to Google.Cloud.Storage.V1.Tests. That's why the build action is not passing I think.
Maybe for now remove the dependencies.

@jskeet we might need to include this one in regenerateProjects?